client/components/common/Tabs/Tabs.jsx

```diff
@@ -1,15 +1,57 @@
 import React, { useState, useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
+import { useLocation, useNavigate } from 'react-router-dom';
 import styles from './Tabs.module.css';
 
 const Tabs = ({ tabs }) => {
-  const [selectedTab, setSelectedTab] = useState(0);
+  const location = useLocation();
+  const navigate = useNavigate();
   const tabRefs = useRef([]);
 
+  const getTabIndexFromQuery = () => {
+    const searchParams = new URLSearchParams(location.search);
+    const tabKey = searchParams.get('tab');
+    if (!tabKey) return 0;
+
+    const index = tabs.findIndex(tab => tab.tabKey === tabKey);
+    return index >= 0 ? index : 0;
+  };
+
+  const [selectedTab, setSelectedTab] = useState(() => {
+    const searchParams = new URLSearchParams(location.search);
+    const tabKey = searchParams.get('tab');
+    if (!tabKey) return 0;
+
+    const index = tabs.findIndex(tab => tab.tabKey === tabKey);
+    return index >= 0 ? index : 0;
+  });
+
   useEffect(() => {
     tabRefs.current = tabRefs.current.slice(0, tabs.length);
   }, [tabs]);
 
+  useEffect(() => {
+    const queryIndex = getTabIndexFromQuery();
+    if (queryIndex !== selectedTab) {
+      setSelectedTab(queryIndex);
+    }
+  }, [location.search, tabs]);
+
+  const updateSelectedTab = index => {
+    setSelectedTab(index);
+    const tab = tabs[index];
+    const tabKey = tab.tabKey || `tab-${index}`;
+    const searchParams = new URLSearchParams(location.search);
```

Comment:
It doesn't look like it was specified in the original feature request but I think it's worth confirming with Matt. I would prefer just regular urls like `reports/aria-features` etc... but I can ask on the original issue.

Reply:
Oh good thought! I like that design better. Should be an easy swap. Can handle it on Monday.